app/src/test/kotlin/com/wire/android/mapper/MessageContentMapperTest.kt

```diff
@@ -158,16 +162,24 @@ class MessageContentMapperTest {
             "image/jpg",
             AssetMetadata.Image(100, 100),
             TestMessage.DUMMY_ASSET_REMOTE_DATA.copy(assetId = "image-id2"),
+            Message.UploadStatus.NOT_UPLOADED,
             Message.DownloadStatus.NOT_DOWNLOADED
         )
-        // When - Then
-        val resultContentOther = mapper.toAsset(QualifiedID("id", "domain"), "message-id", unknownImageFormatMessage)
-        coVerify(exactly = 0) { arrangement.getMessageAssetUseCase.invoke(any(), any()) }
-        assert(resultContentOther is AssetMessage && resultContentOther.assetId.value == unknownImageFormatMessage.remoteData.assetId)
-        // When - Then
-        val resultContentImage = mapper.toAsset(QualifiedID("id", "domain"), "message-id", correctJPGImage)
-        coVerify(exactly = 1) { arrangement.getMessageAssetUseCase.invoke(any(), any()) }
-        assert(resultContentImage is ImageMessage && resultContentImage.assetId.value == correctJPGImage.remoteData.assetId)
+
+        val testMessage1 = buildAssetMessage(unknownImageMessageContent)
+        val testMessage2 = buildAssetMessage(correctJPGImage)
+
+        with(arrangement) {
+            // When - Then
+            val resultContentOther = mapper.toUIMessageContent(AssetMessageData(unknownImageMessageContent), testMessage1, scope)
+            coVerify(exactly = 0) { arrangement.getMessageAssetUseCase.invoke(any(), any()) }
+            assert(resultContentOther is AssetMessage && resultContentOther.assetId.value == unknownImageMessageContent.remoteData.assetId)
+
+            // When - Then
+            val resultContentImage = mapper.toUIMessageContent(AssetMessageData(correctJPGImage), testMessage2, scope)
+            coVerify(exactly = 1) { arrangement.getMessageAssetUseCase.invoke(any(), any()) }
+            assert(resultContentImage is ImageMessage && resultContentImage.assetId.value == correctJPGImage.remoteData.assetId)
```

Comment:
Just a question out of curiosity, why we don't use the `assert` from JUnit framework ? I think that will provide more info in case of errors :D

Reply:
I just extended the way of operating from previous tests, but sure, let's use `assertTrue` instead